src/globus_sdk/_testing/data/flows/run_flow.py

```diff
@@ -20,6 +20,15 @@
         json=TWO_HOP_TRANSFER_RUN,
         match=[matchers.json_params_matcher(params=_request_params)],
     ),
+    # 'lenient' is the same as the 'default' above, but without requiring
+    # payload matching, which makes it more usable for a wide variety of tests
+    # TODO: revisit which of the fixtures should have which param matchers
+    lenient=RegisteredResponse(
```

Comment:
😲

Reply:
Yeah, I have my own opinion here, which is a bit stronger than what I've left in the comments. But for now, I just needed an escape-hatch.